3-JSS/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "2-webpack",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack serve"
```

Comment:
change to `start` the convention for front end projects is to run `npm run start` and this should open the dev env